test/src/main/scala/scala/test/large_classpath/BUILD

```diff
@@ -0,0 +1,910 @@
+load("//scala:scala.bzl", "scala_binary", "scala_library")
+scala_binary(
+    main_class = "scala.test.large_classpath.ObjectWithLargeClasspath",
+    name = "largeClasspath",
+    srcs = ["ObjectWithLargeClasspath.scala"],
+    deps = [
+        ":empty1",
```

Comment:
Please make a helper .bzl file to return both this list of targets, and create the list of libraries below.